applications/visualizer/backend/visualizer/views.py

```diff
@@ -22,3 +23,17 @@ def index(request, path=""):
     except FileNotFoundError:
         return index(request, "")  # index.html
     return FileResponse(fullpath.open("rb"), content_type=content_type)
+
+
+def access_bucket_artifact(request, path):
```

Comment:
question (non-blocking): Am I correct to assume that this strategy requires the bucket to be publicly accessible?

Reply:
Yes, the bucket will always be public, that's why we can adopt this strategy.